### PR TITLE
Add warning msgs when RunId/RunRecord is not found when stopping an Adapter/Program

### PR DESCRIPTION
**Pull request by gokulavasan**

### CODE REVIEW

**Comment by gokulavasan** [comment]
Build : http://builds.cask.co/browse/CDAP-DUT1787-1

File: cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/adapter/AdapterService.java [hunk 1]
```diff
@@ -583,6 +583,8 @@ private void stopWorkerAdapter(Id.Namespace namespace, AdapterDefinition adapter
     if (adapterRun != null) {
       RunId runId = RunIds.fromString(adapterRun.getPid());
       lifecycleService.stopProgram(workerId, runId);
+    } else {
+      LOG.warn("RunRecord not found for Adapter {}", adapterSpec.getName());
```
Comment (jwang47) [inline cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/adapter/AdapterService.java]: `RunRecord not found for Adapter to be stopped`?

File: cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/services/ProgramLifecycleService.java [hunk 2]
```diff
@@ -198,6 +195,8 @@ public void stopProgram(Id.Program programId, RunId runId) throws ExecutionExcep
     ProgramRuntimeService.RuntimeInfo runtimeInfo = runtimeService.lookup(programId, runId);
     if (runtimeInfo != null) {
       runtimeInfo.getController().stop().get();
+    } else {
+      LOG.warn("RunTimeInfo not found for Program {} RunId {}", programId, runId);
```
Comment (jwang47) [inline cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/services/ProgramLifecycleService.java]: `RunTimeInfo not found for Program {} RunId {} to be stopped`?

**Comment by gokulavasan** [comment]
@jwang47 Addressed your comments. Please review.

**Comment by jwang47** [comment]
LGTM

**Pull request merged**